Add the support blog post; minor tweaks to GGSC page;

fubar2 opened the pull request:
This is the first official announcement from the InterGalactic Ministry of Truth :)

The site tag does not seem to be used at the moment, so to tag my crazy deluge of posts, I just made that up. Are there plans to use those header tags?

fubar2 commented:
@bgruening: Now that they are blog posts, I deleted the working text from https://docs.google.com/document/d/1y7P7FKWRE3E4UnaXHLMm_f4tr5LFpRmtzXFvxqxdLCE/edit#heading=h.w8fh1ywdnhfw so it's a little less messy - can replace but seems sensible to only keep things we are working on there?

beatrizserrano commented on content/blog/2022-01-14-users-guide-to-contribution-ross/index.md:
```diff
@@ -1,12 +1,13 @@
 ---
 title: "A contributor's guide to the Galaxy"
-tease: "Contribution opportunities"
+tease: "Contribution opportunities for community members"
```
Do you think that this post is useful only if you're already a member? I'd say that some are good mechanisms to _become_ a member

fubar2 replied:
Yes, maybe best to revert to what it was - will do now

beatrizserrano commented on content/blog/2022-01-14-users-guide-to-contribution-ross/index.md:
```diff
@@ -1,12 +1,13 @@
 source_blog: ""
 source_blog_url: ""
 autotoc: false
+site: InterGalactic Ministry of Truth
```
😆

beatrizserrano commented on content/blog/2022-01-16-open-scientific-computing-ross/index.md:
```diff
@@ -27,8 +34,10 @@ A common problem that prevents many researchers from routinely sharing their wor
+Exactly how that will change science is hard to predict. Negative impacts seem unlikely, other than on   commercial competitor profitability. The peer review process would probably be far more useful, if all the reviewers could readily share, replicate and scrutinise a transparent experimental analysis. It seems likely to speed up the validation of more reliable, replicable new methods and findin ...
 
-One reason to [become involved](../2022-01-14-users-guide-to-contribution-ross) in the Galaxy project is to contribute to normalising open science computing. Exactly how that will change science is hard to predict but we can make some informed guesses. Negative outcomes seem unlikely, other than perhaps for  commercial competitor profits. It seems likely to speed up the validation of more reliabl ...
+If you've managed to read this far, you may want to learn more about opportunities to help make open science computing more accessible. If so, you are cordially invited to [become involved in the Galaxy project](../2022-01-14-users-guide-to-contribution-ross) in any way you can.
```
Could this be a good place to link the contributing page too?

fubar2 replied:
Maybe...it's a little awkward having two parallel resources :) 
Let me ponder it.

fubar2 replied:
@beatrizserrano: ok, reverted. I think we are done for now. The ministry for truth is established.

beatrizserrano replied:
do we merge then?

fubar2 replied:
Both linked now - it's not pretty in some ways but they are different enough to be useful. The blog's not as exhaustive - maybe it needs merging but I'm over it for now.

beatrizserrano replied:
I think that they are addressing different audiences, depending on the reading preferences.

fubar2 replied:
@beatrizserrano: merge please. I'm done for now :)
Thanks!

bgruening commented on content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md:
```diff
@@ -0,0 +1,24 @@
+site: freiburg
+tags: [comment]
+title: Galaxy as a sharing infrastructure and how this enables efficient user support
+authors: "Bjoern Gruening, Cristóbal Gallardo, Ross Lazarus"
```
What about Jen? I don't mind adding many authors that do support this blog.

fubar2 replied:
@bgruening  - sure but who is Jen ? :) Need a name to fix it.

fubar2 replied:
oh - Jen! Sure. I'll add her.

fubar2 commented:
@beatrizserrano 
Fixed a trivial hub landing page syntax error :)
Also fixed Jen's name to the full version on the support blog post - please merge?

beatrizserrano commented:
thank you @fubar2!!
